.gitignore

```diff
@@ -1,2 +1,5 @@
+# dependencies
 /node_modules
-/dist
\ No newline at end of file
+
+# Babel output
```

Comment:
This is not very accurate, as Babel is gone from the deps. Can you just revert all changes this file please?

Reply:
Reverted in 22b43cf8b3594215f9d53d883466d45c7dceff56